src/components/Input/Input.tsx

```diff
@@ -14,18 +14,21 @@ export interface InputProps {
 
   /** Whether the input should be disabled. */
   readonly isDisabled?: boolean;
+
+  /** The element that will be placed on the left side of the input. */
+  readonly leftElement?: ReactNode;
 }
 
 interface Props extends InputProps {
 
   /** Name attribute. */
-  readonly name: string;
+  readonly name?: string;
 
   /** Input value. */
-  readonly value: string;
+  readonly value?: string;
 
   /** Input change callback. */
-  readonly onChange: ChangeEventHandler<HTMLInputElement>;
+  readonly onChange?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
Doesn't `InputHTMLAttributes<HTMLInputElement>` have `onChange` prop? I think we can omit it, also probably `value` and `name` too.